package.json

```diff
@@ -26,15 +25,14 @@
     "tcomb": "^3.0.0"
   },
   "devDependencies": {
+    "@types/eslint-plugin-prettier": "^2.2.0",
+    "@types/prettier": "^1.7.0",
```

Comment:
Just curious, what's this for? `@types/eslint-plugin-prettier`, `@types/prettier`

Reply:
These are Flow types auto-installed by an extension of VSCode. I'll remove them.